### ISSUE TITLE
Use ES5 in npm package

### ISSUE THREAD

**Issue by jiyinyiyong** [tbranyen/diffhtml#67]
The bundled `dist/diffhtml.js` has some problem using in Webpack:

```text
webpack: bundle is now INVALID.
Hash: 206a823a10edb9d87368
Version: webpack 1.13.1
Time: 118ms
                               Asset      Size  Chunks             Chunk Names
                           bundle.js    339 kB       0  [emitted]  main
0.58c44d06e33087563e03.hot-update.js   3.17 kB       0  [emitted]  main
58c44d06e33087563e03.hot-update.json  36 bytes          [emitted]
chunk    {0} bundle.js, 0.58c44d06e33087563e03.hot-update.js (main) 297 kB [rendered]
   [77] ./src/hud.js 3.01 kB {0} [built]
     + 82 hidden modules

WARNING in ./~/diffhtml/dist/diffhtml.js
Critical dependencies:
1:477-484 This seems to be a pre-built javascript file. Though this is possible, it's not recommended. Try to require the original source to get better results.
 @ ./~/diffhtml/dist/diffhtml.js 1:477-484
webpack: bundle is now VALID.
```

Could you release the compiled ES5 to npm please? Adding a `gulpfile.js` and `prepublish` will make it.

**Comment by tbranyen** [comment]
Where is it invalid? Why would you assume that rebuilding would fix the problem? Can you hop into the Gitter channel and we can talk about this in more detail?

**Comment by tbranyen** [comment]
Talked in the Gitter channel about this issue and decided the fix for now is to publish the Babel intermediary CJS files to NPM starting with the version 0.8.3.

**Comment by tbranyen** [comment]
Resolved in 0.8.4! We published the source files necessary under `dist/cjs`.
